<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvPage</class>
 <widget class="QWizardPage" name="AdvPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Advanced Settings</string>
  </property>
  <property name="subTitle">
   <string>Please specify the advanced settings.</string>
  </property>
  <widget class="QLineEdit" name="txtRouteDelay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>227</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtVerbose">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>201</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMute">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>252</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>252</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mute:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>203</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Verbosity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>226</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Route delay:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings for Windows</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MSSFIX:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMsfix">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>21</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox { 
spacing: 5px;
}

QCheckBox::indicator {
width: 13px;
height: 13px;
}

QCheckBox::indicator:unchecked:disabled {
image: url(:/images/styles/checkbox_unchecked_dis.png);
}

QCheckBox::indicator:checked:disabled {
image: url(:/images/styles/checkbox_checked_dis.png);
}

QCheckBox::indicator:unchecked:enabled {
image: url(:/images/styles/checkbox_unchecked.png);
}

QCheckBox::indicator:checked:enabled {
image: url(:/images/styles/checkbox_checked.png);
}

QCheckBox::indicator:unchecked:enabled:hover {
image: url(:/images/styles/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:checked:enabled:hover {
image: url(:/images/styles/checkbox_checked_hover.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbRouteMethod">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>19</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>22</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Route method:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Windows directory:</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="cbWinDirUseDefault">
        <property name="styleSheet">
         <string notr="true">QRadioButton { 
spacing: 5px;
}

QRadioButton::indicator {
width: 13px;
height: 13px;
}

QRadioButton::indicator::unchecked:enabled {
image: url(:/images/styles/radiobutton_unchecked.png);
}

QRadioButton::indicator::checked:enabled {
image: url(:/images/styles/radiobutton_checked.png);
}

QRadioButton::indicator::unchecked:disabled {
image: url(:/images/styles/radiobutton_unchecked_dis.png);
}

QRadioButton::indicator::checked:disabled {
image: url(:/images/styles/radiobutton_checked_dis.png);
}</string>
        </property>
        <property name="text">
         <string>Use default(c:\windows)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="cbWinDirOther">
        <property name="styleSheet">
         <string notr="true">QRadioButton { 
spacing: 5px;
}

QRadioButton::indicator {
width: 13px;
height: 13px;
}

QRadioButton::indicator::unchecked:enabled {
image: url(:/images/styles/radiobutton_unchecked.png);
}

QRadioButton::indicator::checked:enabled {
image: url(:/images/styles/radiobutton_checked.png);
}

QRadioButton::indicator::unchecked:disabled {
image: url(:/images/styles/radiobutton_unchecked_dis.png);
}

QRadioButton::indicator::checked:disabled {
image: url(:/images/styles/radiobutton_checked_dis.png);
}</string>
        </property>
        <property name="text">
         <string>Other</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="cbWinDirEnvironment">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>7</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton { 
spacing: 5px;
}

QRadioButton::indicator {
width: 13px;
height: 13px;
}

QRadioButton::indicator::unchecked:enabled {
image: url(:/images/styles/radiobutton_unchecked.png);
}

QRadioButton::indicator::checked:enabled {
image: url(:/images/styles/radiobutton_checked.png);
}

QRadioButton::indicator::unchecked:disabled {
image: url(:/images/styles/radiobutton_unchecked_dis.png);
}

QRadioButton::indicator::checked:disabled {
image: url(:/images/styles/radiobutton_checked_dis.png);
}</string>
      </property>
      <property name="text">
       <string>Use environment</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="cbWinDirPath">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>27</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QRadioButton { 
spacing: 5px;
}

QRadioButton::indicator {
width: 13px;
height: 13px;
}

QRadioButton::indicator::unchecked:enabled {
image: url(:/images/styles/radiobutton_unchecked.png);
}

QRadioButton::indicator::checked:enabled {
image: url(:/images/styles/radiobutton_checked.png);
}

QRadioButton::indicator::unchecked:disabled {
image: url(:/images/styles/radiobutton_unchecked_dis.png);
}

QRadioButton::indicator::checked:disabled {
image: url(:/images/styles/radiobutton_checked_dis.png);
}</string>
      </property>
      <property name="text">
       <string>Path</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPath">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>25</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">:enabled {border: 1px solid black; background-color: rgb(255, 255, 255);}
:disabled {border: 1px solid black; background-color: rgb(195, 195, 195);}
:hover {border: 1px solid red; }</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>25</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>use \\ instead of \</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cbFloat">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>238</y>
     <width>21</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox { 
spacing: 5px;
}

QCheckBox::indicator {
width: 13px;
height: 13px;
}

QCheckBox::indicator:unchecked:disabled {
image: url(:/images/styles/checkbox_unchecked_dis.png);
}

QCheckBox::indicator:checked:disabled {
image: url(:/images/styles/checkbox_checked_dis.png);
}

QCheckBox::indicator:unchecked:enabled {
image: url(:/images/styles/checkbox_unchecked.png);
}

QCheckBox::indicator:checked:enabled {
image: url(:/images/styles/checkbox_checked.png);
}

QCheckBox::indicator:unchecked:enabled:hover {
image: url(:/images/styles/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:checked:enabled:hover {
image: url(:/images/styles/checkbox_checked_hover.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>256</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comp-LZO:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbUserPass">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>203</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox { 
spacing: 5px;
}

QCheckBox::indicator {
width: 13px;
height: 13px;
}

QCheckBox::indicator:unchecked:disabled {
image: url(:/images/styles/checkbox_unchecked_dis.png);
}

QCheckBox::indicator:checked:disabled {
image: url(:/images/styles/checkbox_checked_dis.png);
}

QCheckBox::indicator:unchecked:enabled {
image: url(:/images/styles/checkbox_unchecked.png);
}

QCheckBox::indicator:checked:enabled {
image: url(:/images/styles/checkbox_checked.png);
}

QCheckBox::indicator:unchecked:enabled:hover {
image: url(:/images/styles/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:checked:enabled:hover {
image: url(:/images/styles/checkbox_checked_hover.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>239</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Float:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbNobind">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>221</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox { 
spacing: 5px;
}

QCheckBox::indicator {
width: 13px;
height: 13px;
}

QCheckBox::indicator:unchecked:disabled {
image: url(:/images/styles/checkbox_unchecked_dis.png);
}

QCheckBox::indicator:checked:disabled {
image: url(:/images/styles/checkbox_checked_dis.png);
}

QCheckBox::indicator:unchecked:enabled {
image: url(:/images/styles/checkbox_unchecked.png);
}

QCheckBox::indicator:checked:enabled {
image: url(:/images/styles/checkbox_checked.png);
}

QCheckBox::indicator:unchecked:enabled:hover {
image: url(:/images/styles/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:checked:enabled:hover {
image: url(:/images/styles/checkbox_checked_hover.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>203</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Auth user/pass:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>223</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>No bind:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbCompLzo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>255</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox { 
spacing: 5px;
}

QCheckBox::indicator {
width: 13px;
height: 13px;
}

QCheckBox::indicator:unchecked:disabled {
image: url(:/images/styles/checkbox_unchecked_dis.png);
}

QCheckBox::indicator:checked:disabled {
image: url(:/images/styles/checkbox_checked_dis.png);
}

QCheckBox::indicator:unchecked:enabled {
image: url(:/images/styles/checkbox_unchecked.png);
}

QCheckBox::indicator:checked:enabled {
image: url(:/images/styles/checkbox_checked.png);
}

QCheckBox::indicator:unchecked:enabled:hover {
image: url(:/images/styles/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:checked:enabled:hover {
image: url(:/images/styles/checkbox_checked_hover.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
